src/shared/selinux-util.c

```diff
@@ -155,7 +155,7 @@ static int mac_selinux_label_post(int dir_fd, const char *path) {
 bool mac_selinux_use(void) {
```

Comment:
i have the suspicion that this function will come to different results now in many cases, since previously we'd call is_selinux_enabled() no matter what, but now we'll only call it after dlopen_selinux() was called, which we quite likely might never do